src/app/modules/ix-forms/components/ix-list/ix-list-item/ix-list-item.component.scss

```diff
@@ -9,6 +9,10 @@
   width: 100%;
 }
 
+.list-item-content {
```

Comment:
This is okay, but this will break other components that use ix-list and had margins added manually.